tests/test_deprecations.py

```diff
@@ -8,13 +8,9 @@
 
 # New modules
 import rasterio
-from rasterio import windows
```

Comment:
@eseglem this isn't deprecated yet, so it should not get grouped in the following list under the comment.